drivers/sensor/ens210/ens210.c

```diff
@@ -234,7 +275,9 @@ static int ens210_init(struct device *dev)
 		return -EIO;
 	}
 
-	/* Set continuous measurement */
+	ens210_sys_enable(drv_data->i2c, (ENS210_T_RUN | ENS210_H_RUN) != 1);
+
+	/* Set measurement mode*/
 	ret = i2c_reg_write_byte(drv_data->i2c, DT_INST_0_AMS_ENS210_BASE_ADDRESS,
```

Comment:
We only need to write ENS210_REG_SENS_RUN if one of the sensors is in continuos mode.